test/Interop/Cxx/value-witness-table/witness-layout-opts-irgen.swift

```diff
@@ -0,0 +1,20 @@
+// RUN: %target-swift-emit-ir %s -I %S/Inputs -enable-cxx-interop | %FileCheck %s
+
+import WitnessLayoutOpts
+
+protocol Dummy { }
+
+// CHECK-DAG: @"$sBi8_WV" = external global i8*, align 8
```

Comment:
Would be nice to add a short comment that this test checks that the compiler uses well-known witness tables for types that have the appropriate shape.

Reply:
Will do.